src/Service.GraphQLBuilder/Queries/InputTypeBuilder.cs

```diff
@@ -53,27 +62,12 @@ private static List<InputValueDefinitionNode> GenerateOrderByInputFieldsForBuilt
                             new List<DirectiveNode>())
                         );
                 }
-                else
```

Comment:
as per https://github.com/Azure/project-hawaii/blob/main/rfcs/graphql.md#sorting, this should be supported. We should add a discussion idea, to check how many upvotes do we get that are interested in this feature